classes/book.rb

```diff
@@ -0,0 +1,15 @@
+# Book class with title and author attributes
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals << rental
+  end
```

Comment:
- [ ] Great job so far, in order to achieve the` has-many` association the book itself needs to be sent. Kindly fix that.

`i.e  Rental.new(....)`

Reply:
Done